src/Utils/Notification.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import Snackbar from 'material-ui/Snackbar';
+import IconError from "material-ui/svg-icons/alert/error-outline";
+import IconSuccess from "material-ui/svg-icons/navigation/check";
+import {css} from 'aphrodite';
+import AppStyles from "./../AppStyles";
+import config from "./../config/config"
+
+class ConfirmationMessage extends Component {
+	
+	constructor(props) {
+		super(props);
+	}
+
+	getColorForMode() {
```

Comment:
I don't like the hard-coded stuff here. It would be better to pass the color and the icon as props to the component instead of a mode. There might be more modes.